src/main/java/org/apache/commons/collections4/IteratorUtils.java

```diff
@@ -127,7 +87,7 @@ private IteratorUtils() {}
      * @return an iterator over nothing
      */
     public static <E> ResettableIterator<E> emptyIterator() {
-        return EmptyIterator.<E>resettableEmptyIterator();
+        return EmptyIterator.resettableEmptyIterator();
```

Comment:
These changes are unrelated, small PRs are better for reviewers ;-)

Reply:
Done.